### ISSUE TITLE
Setting options from a blueprint mistakenly save PHP serialized data as a string

### ISSUE THREAD

**Issue by meitar** [bradp/vv#247]
Some WordPress settings are not simple strings, but rather serialized PHP data structures. I'd like to be able to use VV's Blueprint file to configure these values, but currently VV only recognizes string literals as option values.

For example, many popular plugins (like the Simple Custom CSS plugin), store its value in the database as a serialized array, like this:

```
mysql> select * from wp_options WHERE option_name LIKE '%sccss%'\G
*************************** 1. row ***************************
   option_id: 124
 option_name: sccss_settings
option_value: a:1:{s:13:"sccss-content";s:34:".site-title { font-size: larger; }";}
    autoload: yes
1 row in set (0.00 sec)
```
Unfortunately, using a blueprint file like this:
```
"options": [
    "sccss_settings::a:1:{s:13:\"sccss-content\";s:34:\".site-title { font-size: larger; }\";}"
]
```
results in the plugin's option being set to a `string` as opposed to a serialized `array`.

**Comment by meitar** [comment]
I've opened PR #248 to add this capability.

**Comment by meitar** [comment]
This is fixed now that #248 is merged.
